application/app/src/main/java/com/example/buber/Views/Fragments/AcceptTripRequestFragment.java

```diff
@@ -95,10 +113,33 @@ public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
             viewContactButton.setOnClickListener(v -> {
                 handleViewRiderContact(tripSearchRecord.getRiderID());
             });
-        }
 
+            //finds and initializes the mapView fragment
+            MapView mapView = view.findViewById(R.id.lite_map);
+            MapsInitializer.initialize(getActivity());
+            mapView.onCreate(savedInstanceState);
+            mapView.onResume();
+
+            //Gets the mapAsync and overlays the route polyline and start/end markers
+            mapView.getMapAsync(googleMap -> {
+                LatLng origin = tripSearchRecord.getStartLatLng();
+                LatLng destination = tripSearchRecord.getEndLatLng();
+                new GetPathFromLocation(origin, destination, polyLine -> {
+                    routePointList = polyLine.getPoints();
+                    googleMap.addPolyline(polyLine);
+                }).execute();
+                googleMap.addMarker(new MarkerOptions().position(origin).title("start"));
+                googleMap.addMarker(new MarkerOptions().position(destination).title("end"));
+                LatLngBounds.Builder boundsBuilder = new LatLngBounds.Builder();
+                boundsBuilder.include(origin);
+                boundsBuilder.include(destination);
+                LatLngBounds latLngBounds = boundsBuilder.build();
+                googleMap.moveCamera(CameraUpdateFactory.newLatLngZoom(latLngBounds.getCenter(),10));
```

Comment:
For me the map fragment is always opening in lite mode(showing the whole world). I zoomed in on the route a bit to make it easier

Reply:
I made the map in lite mode to make the fragment less expensive - I figured it should only be an image of the route because its just preview for the driver before they accept the trip.

The displaying the whole world thing is strange though. It wasn't doing that for me.